Enables support for ext4, the "load" command, and using bootz with raw initrds.

--- a/configs/mx53loco_defconfig
+++ b/configs/mx53loco_defconfig
@@ -21,6 +21,7 @@
 CONFIG_CMD_MII=y
 CONFIG_CMD_PING=y
 CONFIG_CMD_EXT2=y
+CONFIG_CMD_EXT4=y
 CONFIG_CMD_FAT=y
 CONFIG_CMD_FS_GENERIC=y
 CONFIG_ENV_OVERWRITE=y
